api/src/test/java/org/pmiops/workbench/api/DisksControllerTest.java

```diff
@@ -202,13 +206,29 @@ public void test_listOwnedDisksInWorkspace() throws ApiException {
     // GCE Disk: 3 disks in total, 2 are active, newer one is inactive, returns the most recent
     // active ones.
     LeonardoListPersistentDiskResponse olderGceDisk =
-        newListPdResponse(
-            user.generatePDName(), LeonardoDiskStatus.READY, NOW.minusMillis(200).toString(), null);
+        createLeonardoListPersistentDiskResponse(
```

Comment:
what do you think about adding a specialized form to clarify that it's intended for runtime-PDs instead of app-PDs?  Something like
```
createLeonardoListRuntimePDResponse(...params without appType...) {
  return createLeonardoListPersistentDiskResponse(...params, plus appType=null...);
}
```